services/QuillLMS/spec/queries/quill_big_query/schools_containing_certain_teachers_query_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module QuillBigQuery
+  describe SchoolsContainingCertainTeachersQuery do
+    context 'for schools all time', :big_query_snapshot do
+      include_context 'QuillBigQuery TestRunner Setup'
+
+      let(:schools) { create_list(:school, 10) }
+      let(:chosen_school_one) { create(:school) }
+      let(:chosen_school_two) { create(:school) }
```

Comment:
Would love to see the addition of a `let(:excluded_school) { create(:school) }` that gets included in `cte_records` just to make sure that our filtering is actually doing what we want and not just returning all schools.